fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt

```diff
@@ -81,22 +85,54 @@ fun ReviewQualityCheckScaffold(
 
 @Composable
 private fun Header() {
+    val betaBorderColor: Color
+    val betaTextColor: Color
+    if (isSystemInDarkTheme()) {
+        betaBorderColor = PhotonColors.LightGrey10
+        betaTextColor = FirefoxTheme.colors.textActionPrimary
+    } else {
+        betaBorderColor = FirefoxTheme.colors.actionTertiary
+        betaTextColor = FirefoxTheme.colors.textSecondary
+    }
+
```

Comment:
I confirmed with Kate that the text & border colors here are one-off and intentionally do not have a matched-up color token.

Reply:
Should we create a bug to track this, so going forward we can align?

Reply:
Not in this scenario. This won't be conforming to an Acorn token. There may be, long term, "one-off" tokens we make at a feature scope for Fenix-only that are provided by the figma designer (i.e. not from the Acorn system), but those are very very early conversations.